app/(afterLogin)/(employeeInformation)/employees/manage-employees/[id]/_components/job/addEmployeeJobInfrmation/index.tsx

```
@@ -6,47 +6,96 @@ import WorkScheduleForm from '../../../../_components/allFormData/workScheduleFo
         handleClose();
-        onInfoSubmition && onInfoSubmition();
+        
+        // Call the callback to refresh job information data
+        if (onJobInfoUpdated) {
+          setTimeout(() => {
+            onJobInfoUpdated();
+          }, 500);
+        }
+      },
+      onError: (error: any) => {
+        console.error('Error creating job information:', error);
```

ESLint

[error] Unexpected console statement.

(no-console)